yt/utilities/lib/pixelization_routines.pyx

```diff
@@ -668,6 +668,26 @@ def pixelize_aitoff(np.float64_t[:] azimuth,
         xmax = fmax(xmax, x)
         ymin = fmin(ymin, y)
         ymax = fmax(ymax, y)
+        # special cases where the projection of the cell isn't
+        # bounded by the rectangle (in image space) that bounds its corners.
+        # Note that performance may take a serious hit here. The overarching algorithm
```

Comment:
Just to follow up on this statement, do you think that in the typical, previous optimization case, this will be a problem?

Reply:
No, I couldn't measure any difference in performance for the more typical use case.